src/spec.cr

```diff
@@ -23,6 +23,25 @@ module Shards
       end
     end
 
+    class Library
+      property soname : String
```

Comment:
Let's keep `soname`, because keeping `name` for `pkg-config` is a _very_ good idea.